minutes/README.md

```diff
@@ -0,0 +1,4 @@
+A directory containing the minutes from previous hangouts.
+
+Future hangouts have an agenda at https://hackpad.com/conda-forge-meeting-notes-WZIa4PBQ6sz.
```

Comment:
Would it be worthwhile to include the link in the top level directory too? We might also want to mention meeting time somewhere.